test/get-authentication-context.test.js

```diff
@@ -14,16 +14,16 @@ test('getAuthenticationContext returns undefined when lp_auth cookie is set to a
 })
 
 test('getAuthenticationContext returns undefined when lp_auth cookie does not contain a token', () => {
-  Cookies.set('lp_auth', {})
+  Cookies.set('lp_auth', JSON.stringify({}))
```

Comment:
In v3, `Cookies.set` no longer implicitly calls stringify. We aren't using this method in the code and we weren't using `toJson()` either, so it doesn't affect any source code.

https://github.com/js-cookie/js-cookie/releases/tag/v3.0.0

Reply:
Sounds good. Any value in creating a testing `util` function/interface to `Cookies` that handles the `JSON.stringify`. There are 3 examples here 😏 (I'm good on holding off too but figured I would at least mention it)

Reply:
I could be convinced if the source code was using it, but I'm pretty ok with being repetitive in tests!